CMakeLists.txt

```diff
@@ -106,19 +106,20 @@ endif()
 find_package(Freetype REQUIRED)
 
 if(freetype-gl_WITH_GLEW)
-    add_definitions(-DWITH_GLEW=1)
+    set(FREETYPE_GL_USE_GLEW 1)
```

Comment:
What's the difference between `set` and `add`?

Reply:
`add_definitions` add a predefined macro through add parameters to compiler, such as `gcc ... -DXXX=xxx`

`set` set a cmake variable, and the cmake variable is used by `configure_file` at line 134. Then macro writed to a header file `config.h` by `configure_file`, rather than add parameter to compiler.

`configure_file` generate `config.h` with a template `config.h.in` at my code.

Reply:
+1